### PR TITLE
Update dependencies, consolidate versions of dependencies to parent gradle

### PR DESCRIPTION
**Pull request by barnhill**

### CODE REVIEW

**Comment by chrisjenx** [comment]
Feel free to rebase this.

**Comment by barnhill** [comment]
I'll do that as soon as I get home

**Comment by barnhill** [comment]
Done ... thanks for taking a look and let me know if I can help out any.

File: CalligraphySample/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.3"
+    compileSdkVersion rootProject.ext.compileSdkVersion
```
Comment (chrisjenx) [inline CalligraphySample/build.gradle]: So you know that as these are in the `allProject` closure, you don't need to define `rootProject.ext.`

**Comment by barnhill** [comment]
I'll fix them

**Pull request merged**